app/controllers/guest_controller.rb

```diff
@@ -0,0 +1,17 @@
+class GuestsController < ApplicationRecord
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def update
```

Comment:
We aren't exactly updating the guest, but instead creating a new record in the GuestRooms table. You wouldn't need this action in this controller.

The action, `create` would be defined in your `GuestRoomsController` with something like this:

```ruby
def create
  guest = Guest.find(params[:id]) # or guest = Guest.find(params[:guest_id]), depending on how you set up your routes
  room = Room.find(params[:room_id])

  GuestRoom.create!(guest_id: guest.id, room_id: room.id)

  redirect_to "/guests/#{guest.id}"
end
```